sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -71,15 +85,15 @@ class SnackBar extends Component {
     ];
     if (actions != null)
       children.addAll(actions);
-
     return new SlideTransition(
```

Comment:
We used to avoid it with reflection, but that's too high a cost (and was actually quite confusing).
@abarth is noodling with a new approach to this which could work (where all the settings are in a single object, so you just copy the new object over). It'd be expensive to transition to it, but might be worth it.

In what you describe above, is SlideInTransition() still a Widget? Or are transitions a separate class hierarchy?